wiki/Reporting_bad_behaviour/en.md

```diff
@@ -8,8 +8,9 @@ Accidental reports are ignored. Users who deliberately send invalid reports can
 
 | Nature of infringement | Point of contact |
 | :-- | :-- |
-| Offensive private messages | **None**: [ignore the user](/wiki/Client/Interface/Chat_console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Client/Options). |
-| Infringements related to modding and/or mapping | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or members of the [GMT](/wiki/People/The_Team/Global_Moderation_Team) focusing on mapping/modding community moderation, via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
+| Scamming/Phishing in private messages | send a regular report |
+| Other offensive/inappropriate private messages | **None**: [ignore the user](/wiki/Client/Interface/Chat_console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Client/Options) or block them on the website with the button located on their user profile. |
+| Inappropriate conduct from a [BN](/wiki/en/People/The_Team/Beatmap_Nominators) | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
```

Comment:
the bn website covers much more than that, and while I agree we should take "inappropriate behaviour" out from the table, there's still the rest 
![image](https://user-images.githubusercontent.com/4686101/165113527-49b420c6-b6df-4ad5-9a3a-961488b79272.png)